src/translations/da_DK.json

```diff
@@ -304,5 +304,6 @@
   "OFFER_BUNDLE_SELECTOR_DESCRIPTION": "",
   "OFFER_APP_PROMOTION_HEADING": "En forsikringsapp, der løser dine problemer",
   "OFFER_APP_PROMOTION_PARAGRAPH": "",
-  "WEB_OFFER_DOCUMENTS_SECTION_TITLE": "Vigtige dokumenter"
+  "WEB_OFFER_DOCUMENTS_SECTION_TITLE": "Vigtige dokumenter",
+  "STARTPAGE_UNINSURED_BODY_WITHOUT_ACCIDENT": "Beregn prisen på en ny indboforsikring hos Hedvig"
```

Comment:
This text key is actually only used for Sweden so there's no need for a Danish translation. That said, I still think we can merge this since this text key is only temporary and should be removed when we launch accident SE right?

Reply:
Yes that's how I also understand it :)